src/styles/globals.css

```
@@ -243,7 +259,9 @@
 @import 'tw-animate-css';
 @import 'shadcn/tailwind.css';
 @plugin "@tailwindcss/typography";
-@import '@fontsource-variable/geist';
+@import '@fontsource-variable/inter';
```

Biome (2.4.6)

[error] This `@import` is in the wrong position.

(lint/correctness/noInvalidPositionAtImportRule)